orchestration/dap_orchestration/solids.py

```diff
@@ -232,7 +232,8 @@ def eols_transform_records(config: dict[str, str]) -> dict[str, str]:
         "output_dir": String,
     }
 )
-def write_outfiles(context: AbstractComputeExecutionContext, fan_in_results: list[DapSurveyType]) -> None:
+def write_outfiles(context: AbstractComputeExecutionContext,
+                   fan_in_results: list[DapSurveyType]) -> tuple: # todo: can I somehow return "list[DapSurveyType], str"?
```

Comment:
🤔 A dataclass would work here:

```
@dataclass
class FanInResultsWithTsvDir:
     fan_in_results: list[DapSurveyType]
     tsv_dir: str
```
and then just return an instance of that.

Reply:
Added the above dataclass to dap_orchestration.types